### PR TITLE
[MINOR][DOCS] Fix a typo for a configuration property of resources allocation

### PR DESCRIPTION
**Pull request by sarutak**
### What changes were proposed in this pull request?

This PR fixes a typo for a configuration property in the `spark-standalone.md`.
`spark.driver.resourcesfile` should be `spark.driver.resourcesFile`.
I look for similar typo but this is the only typo.

### Why are the changes needed?

The property name is wrong.

### Does this PR introduce _any_ user-facing change?

Yes. The property name is corrected.

### How was this patch tested?

I confirmed the spell of the property name is the correct from the property name defined in o.a.s.internal.config.package.scala.

### CODE REVIEW

File: docs/spark-standalone.md [hunk 1]
```diff
@@ -359,7 +359,7 @@ Spark Standalone has 2 parts, the first is configuring the resources for the Wor
...
 
-The second part is running an application on Spark Standalone. The only special case from the standard Spark resource configs is when you are running the Driver in client mode. For a Driver in client mode, the user can specify the resources it uses via <code>spark.driver.resourcesfile</code> or <code>spark.driver.resource.{resourceName}.discoveryScript</code>. If the Driver is running on the same host as other Drivers, please make sure the resources file or discovery script only returns resources that do not conflict with other Drivers running on the same node.
+The second part is running an application on Spark Standalone. The only special case from the standard Spark resource configs is when you are running the Driver in client mode. For a Driver in client mode, the user can specify the resources it uses via <code>spark.driver.resourcesFile</code> or <code>spark.driver.resource.{resourceName}.discoveryScript</code>. If the Driver is running on the same host as other Drivers, please make sure the resources file or discovery script only returns resources that do not conflict with other Drivers running on the same node.
```
Comment (dongjoon-hyun) [inline docs/spark-standalone.md:362]: Ur, this looks like not a typo, isn't it? So, is this about changing to CamelCase, `resourcesfile` -> `resourcesFile`, only? Is Apache Spark case sensitive?

Comment (sarutak) [reply docs/spark-standalone.md:362]: I think it's not just about notation. Property names in Apache Spark should be case sensitive.

Comment (dongjoon-hyun) [reply docs/spark-standalone.md:362]: Got it. Thanks.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master/3.0.

**Pull request closed without merging**